src/fishbus.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>

```

Comment:
Doesn't this removes support for .NET Core 3.1?

Reply:
Yes, it would. Microsoft wants to make net5.0 the standard for [code sharing moving forward](https://devblogs.microsoft.com/dotnet/the-future-of-net-standard/). On second thought a change like this might be too rushed. Maybe better to start with netstandard2.1?